pubtools/exodus/_tasks/push.py

```diff
@@ -14,48 +14,51 @@ def add_args(self):
         super(ExodusPushTask, self).add_args()
 
         self.parser.add_argument(
-            "--source",
-            help="Local path to a file or directory for sync to the exodus CDN",
-        )
-        self.parser.add_argument(
-            "--dest",
-            help="Remote destination for sync",
+            "--push-item",
+            action="append",
+            help=(
+                """usage: --push-item SRC,DEST"""
```

Comment:
Does this render correctly when you run the command with `--help` ?  I haven't tried it myself but due to the lack of newline in the string, I would expect it to show up mangled like this:

```
usage: --push-item SRC,DESTWhere source is a local path ...
```